### PR TITLE
Typos and params declaration (doxygen)

### PR DESCRIPTION
**Pull request by couet**
- Some classes data member where not properly document (doxygen)
- many typos

### CODE REVIEW

File: math/mathcore/inc/Math/StdEngine.h [hunk 1]
```diff
@@ -54,32 +54,32 @@ namespace ROOT {
...
+
+
+      /**
           @ingroup Random
-          Class to wrap engines fron the C++ standard random library in 
-          the ROOT Random interface. 
-          This casess is then by used by the generic TRandoGen class 
+          Class to wrap engines fron the C++ standard random library in
+          the ROOT Random interface.
+          This cases is then by used by the generic TRandoGen class
```
Comment (ferdymercury) [inline math/mathcore/inc/Math/StdEngine.h:63]: These cases ? This case ? This class?

Comment (ferdymercury) [reply math/mathcore/inc/Math/StdEngine.h:63]: TRandoGen --> TRandomGen ?

File: math/mathcore/inc/Math/RootFinder.h [hunk 2]
```diff
@@ -40,7 +40,7 @@
 /**
    @defgroup RootFinders One-dimensional Root-Finding
    Classes implementing algorithms for finding the roots of a one-dimensional function.
-   Various implementation esists in MathCore and MathMore
+   Various implementation exists in MathCore and MathMore
```
Comment (ferdymercury) [inline math/mathcore/inc/Math/RootFinder.h:43]: implementations exist ?

**Comment by ferdymercury** [comment]
One trick would be to set: WARN_NO_PARAMDOC to YES in the Doxyfile, to find all undocumented params

**Review by lmoneta — APPROVED** [review]
LGTM!
Thank you Olivier for all these fixes

**Comment by couet** [comment]
@ferdymercury 
> One trick would be to set: WARN_NO_PARAMDOC to YES in the Doxyfile, to find all undocumented params

That was not the point. This fixes properly document the class members (sorry "parameters" was not the right word) with the `///<` tag.

**Pull request merged**